interpreter/code_interpreters/languages/applescript.py

```diff
@@ -5,8 +5,10 @@ class AppleScript(SubprocessCodeInterpreter):
     file_extension = "applescript"
     proper_name = "AppleScript"
 
-    def __init__(self):
-        super().__init__()
+    def __init__(self, sandbox: bool):
+        if sandbox:
+            raise NotImplementedError("Sandboxing is not supported for AppleScript.")
```

Comment:
The prompts will maybe need some adjusting, so this doesn't happen. Everything around os, should be checked.